.evergreen.yml

```diff
@@ -14,6 +14,47 @@ variables:
       bucket: mciuploads
       permissions: public-read
       content_type: application/octet-stream
+  - &compass-env
+      CI: "1"
+      EVERGREEN: "1"
+      EVERGREEN_AUTHOR: ${author}
+      EVERGREEN_BRANCH_NAME: ${branch_name}
+      EVERGREEN_BUILD_ID: ${build_id}
+      EVERGREEN_BUILD_VARIANT: ${build_variant}
+      EVERGREEN_EXECUTION: ${execution}
+      EVERGREEN_IS_PATCH: ${is_patch}
+      EVERGREEN_PROJECT: ${project}
+      EVERGREEN_REVISION: ${revision}
+      EVERGREEN_TASK_ID: ${task_id}
+      EVERGREEN_TASK_NAME: ${task_name}
+      EVERGREEN_TASK_URL: https://evergreen.mongodb.com/task/${task_id}
+      EVERGREEN_VERSION_ID: ${version_id}
+      EVERGREEN_WORKDIR: ${workdir}
+      NODE_JS_VERSION: "14.17.5"
+      NPM_VERSION: "7"
+      # secrets
```

Comment:
Here I have one pool of compass env vars used by all the shell.exec (or potentially in future subprocess.exec) scripts. We might want to segment it, though? I suppose some only get used by one task.

But we'll probably be refactoring this whole file soon as we split things into smaller tasks, so maybe not worth overthinking it for now.